packages/edit-widgets/package.json

```diff
@@ -33,6 +33,7 @@
 		"@wordpress/core-data": "file:../core-data",
 		"@wordpress/data": "file:../data",
 		"@wordpress/data-controls": "file:../data-controls",
+		"@wordpress/edit-navigation": "file:../edit-navigation",
```

Comment:
I believe we should also remove this from the dependencies :)

Reply:
True. Forgot it... Sorry.

Reply:
Plus from the lock file and we are good to go, thank you again for leading those efforts 💯

Reply:
Just committed them. Sorry 😅